applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy_runge_kutta_4.h

```diff
@@ -331,7 +331,6 @@ class CompressibleNavierStokesExplicitSolvingStrategyRungeKutta4 : public Explic
         // These will be used in the next RK substep residual calculation to compute the subscales
         auto& r_model_part = BaseType::GetModelPart();
         auto& r_process_info = r_model_part.GetProcessInfo();
-        r_process_info[TIME_INTEGRATION_THETA] = r_process_info[RUNGE_KUTTA_STEP] == 1 ? 0.0 : 0.5;
```

Comment:
Maybe I'm missing something, but this variable needs to be set for the evaluation of the residual. Is it now set in the base class?

Reply:
Now I see it below. Forget about my comment.